### ISSUE TITLE
fix a11y on button and badge

### ISSUE THREAD

**Issue by kevanantha** [evilfactorylabs/anggun#38]
**Describe the bug**
a11y failed if a variant of button/badge **outline** and the bg/type **warning**

**Expected behavior**
fix the a11y

**Screenshots**
![image](https://user-images.githubusercontent.com/31406355/80915738-ce083580-8d86-11ea-8389-b207dd68d385.png)

**Additional context**
https://anggun.now.sh/?path=/story/button--button

see PR: #34
